Source/CartingService.API/Program.cs

```diff
@@ -44,8 +46,11 @@
 
 var app = builder.Build();
 app.UseMiddleware<ExceptionMiddleware>();
+app.UseJwtLoggingMiddleware();
```

Comment:
Do we still need JwtLoggingMiddleware since it's already a part of AuthenticationServiceConfiguration?

Reply:
Ty for the comment, but no, It used only in one place 
![image](https://github.com/PetkevichVladislav/AdvancedDotNetMentoring/assets/87666101/e5c92cb3-9e8c-4af6-ac8c-6e7ea87f2600)